content/release-10-15-0/streaming-analytics-10-15-0-bundle/10_15_0.md

```diff
@@ -10,6 +10,13 @@ This release of Cumulocity IoT Streaming Analytics includes the Apama version 10
 See also [What's New In Apama 10.15.0](https://documentation.softwareag.com/pam/10.15.0/en/webhelp/pam-webhelp/index.html#page/pam-webhelp%2Fco-WhaNewInApa_10150_top.html)
 in the Apama documentation.
 
+### Improvements for multi-tenancy
+
+It is now possible to enable multi-tenant support for your EPL apps.
```

Comment:
This is not true :(

Reply:
EPL apps still does not support multi-tenant. This release we added support for custom multi-tenant Apama microservice. So something like:

Apama custom microservice can now be multi-tenant.